### ISSUE TITLE
FileExistsError: [Errno 17] File exists: '.cache/plugin/git-committers'

### ISSUE THREAD

**Issue by squidfunk** [ojacques/mkdocs-git-committers-plugin-2#11]
I've test-driven version 0.4.2 – it works the first time, but on subsequent invocation of `mkdocs serve`, it errors with:

```
...
INFO     -  git-committers: saving authors cache file
FileExistsError: [Errno 17] File exists: '.cache/plugin/git-committers'
```

I inspected the source code of the plugin, but I'm not sure where the error is coming from. Any idea?

**Comment by ojacques** [comment]
Fixed in 0.4.3. Thank you for the report!

**Comment by squidfunk** [comment]
Perfect! Thanks for the quick fix!
